codespell_lib/data/dictionary_en-GB_to_en-US.txt

```diff
@@ -111,6 +113,8 @@ sanitise->sanitize
 sanitised->sanitized
 serialisation->serialization
 serialise->serialize
+skilful->skillful
```

Comment:
This should be 
```suggestion
skillfull->skillful
```

Reply:
No. skilful is the BE variant.

Reply:
:facepalm:

Reply:
I think I got confused by your typo up the top...